host/xtest/regression_4000.c

```diff
@@ -4214,12 +4236,32 @@ static void xtest_tee_test_4006(ADBG_Case_t *c)
 					priv_key_handle)))
 				goto out;
 
+			num_algo_params = 0;
+			if (tv->algo == TEE_ALG_RSAES_PKCS1_OAEP_MGF1_SHA1 ||
+			    tv->algo == TEE_ALG_RSAES_PKCS1_OAEP_MGF1_SHA256) {
+				algo_params[0].attributeID =
+					TEE_ATTR_RSA_OAEP_MGF_HASH;
+				algo_params[0].content.ref.length =
+					sizeof(sha1_algo_id);
+				algo_params[0].content.ref.buffer =
+					&sha1_algo_id;
+				num_algo_params = 1;
+			}
+
 			priv_key_handle = TEE_HANDLE_NULL;
+			res = ta_crypt_cmd_asymmetric_decrypt(c, &session, op,
+					algo_params, num_algo_params, tv->ctx,
+					tv->ctx_len, out, &out_size);
+
+			if (tv->algo == TEE_ALG_RSAES_PKCS1_OAEP_MGF1_SHA256 &&
+			    res == TEEC_ERROR_NOT_SUPPORTED) {
+				Do_ADBG_Log("RSA-OAEP with a different MGF1 "
+					    "hash not supported: skip "
+					    "subcase");
+				goto skip;
```

Comment:
`break` is enough IMHO.

Reply:
Ack. I'll push an update with these addressed.